runtime/tutor

```diff
@@ -1340,6 +1340,116 @@ actions available in a popup. This will allow you to:
 
 
 
+=================================================================
+=                  CHAPTER 13.1 CREATE NEW SPLIT                =
+=================================================================
+
+In normal mode, press ctrl-w (i.e. start pressing ctrl, and
+while it is pressed, press w) to open the Window menu. This will
```

Comment:
```suggestion
In normal mode, press ctrl-w to open the Window menu. This will
```
I think users know what ctrl-w does, you could remove these extra instructions throughout the rest of this PR too.